tests/unit_tests/libs/controllers/test_predictor.py

```diff
@@ -12,6 +12,7 @@
 from sklearn import preprocessing
 from sklearn.linear_model import LinearRegression
 from sklearn.metrics import r2_score, f1_score, accuracy_score
+from sklearn.datasets import load_iris
```

Comment:
Hmh, never thought about using datasets from sklearn, they do pose some issues with e.g. data being pre encoded I believe, but in this case I guess it works just fine.